### ISSUE TITLE
Win 10 Clipboard History

### ISSUE THREAD

**Issue by maikxmh** [IJHack/QtPass#509]
https://support.microsoft.com/en-us/help/4028529/windows-10-clipboard

starting with Windows 10 1809 there is a new feature -> Clipboard history
all the copied data will be stored, so the "clear clipboard" feature of QtPass doesn't help.

May there is a way to prevent storing the password in clipboard history?

**Comment by Glyxbaer** [comment]
This seems like a rather fundamental issue:
As all processes on the system have access to the clipboard, I think a better approach would be to have an autotype feature for the currently selected entry in QtPass? 

Keepass for example supports both: The deletion of the password history in Win10 and an autotype feature.
![image](https://user-images.githubusercontent.com/902690/74539907-29bce280-4f3f-11ea-9ed0-cf75b0927c0c.png)

Both features would make a great addition to QtPass.

**Comment by annejan** [comment]
I have no Windows machines, so if someone wants to code this up I'm more than willing to test 😄

**Comment by AlexanderPavlenko** [comment]
Also KDE Connect may sync clipboard to all paired devices, and their clipboards doesn't get reset. Maybe "clear clipboard" should also override clipboard with some space-filled string.
+1 to autotype feature without copy-pasting

**Comment by Konfekt** [comment]
There is the [`secret` option](https://github.com/palortoff/wclip/blob/dee2147f3187d555c76161e0d9e8b8a11c1dc472/src/Clipboard.cpp#L27) (used by [wclip](https://github.com/palortoff/wclip/)) that hides the clipboard content from many clipboard history tools, like Aceme, Ditto, ..

**Comment by annejan** [comment]
Windows clipboard history is a Windows feature, not QtPass. Closing.

**Comment by Konfekt** [comment]
True. Any feature of QtPass to avoid the clipboard, such as autotype, already addresses this issue.
